controllers/authorino_controller_test.go

```diff
@@ -146,9 +147,9 @@ var _ = Describe("Authorino controller", func() {
 			Expect(deployment.Spec.Selector.MatchLabels).Should(HaveKeyWithValue("authorino-resource", authorinoInstance.Name))
 			Expect(deployment.Spec.Template.Labels).Should(HaveKeyWithValue("authorino-resource", authorinoInstance.Name))
 			for _, container := range deployment.Spec.Template.Spec.Containers {
-				if container.Name == authorinoContainerName {
+				if container.Name == reconcilers.AuthorinoContainerName {
```

Comment:
The `controller` package no longer knows about the details of how a Deployment is built. Therefore, this kind of assertion now belongs to a unit test of the `reconcilers` IMO.

Reply:
yeah taking a second look I think these tests would better fit in the authroino_reconiler_test.go. Ill try refactor these testcases some time next week.

Reply:
As discussed with Gui a follow-up task will be opened for this. This will leave the PR with a reasonable size and ensure the same functionality remains.

Reply:
Created Follow up task -> https://github.com/Kuadrant/authorino-operator/pull/253